common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -293,49 +300,53 @@ div.video {
           font-size: em(14);
         }
 
-        li {
+        a {
+          @extend %video-button;
           float: left;
-          margin-bottom: 0;
+          background-image: url('../images/vcr.png');
+          background-position: 15px 15px ;
+          background-repeat: no-repeat;
+          border-left: none;
+          box-shadow: 1px 0 0 #555;
+          padding: 0 lh(.75);
+          width: 14px;
 
-          a {
-            @extend %video-button;
-            background-image: url('../images/vcr.png');
-            background-position: 15px 15px ;
-            background-repeat: no-repeat;
-            border-left: none;
-            box-shadow: 1px 0 0 #555;
-            padding: 0 lh(.75);
-            width: 14px;
-
-            &:focus {
-              @extend %ui-depth4;
-              position: relative;
-              outline: $white dotted thin;
-              outline-offset: -2px;
-            }
+          &:focus {
+            @extend %ui-depth4;
+            position: relative;
+            outline: $white dotted thin;
+            outline-offset: -2px;
+          }
 
-            &:empty {
-              height: 46px;
-              background-position: 15px 15px;
-            }
+          &:empty {
+            height: 46px;
+            background-position: 15px 15px;
+          }
 
-            &.play {
-              background-position: 17px -114px;
-            }
+          &.play {
```

Comment:
> NIT: We already use the class .play, so this is just a small nit, but having a stateful class label such as "is-playing" or "is-paused" is a small nice to have.

I don't think that it is a part of this PR. It might require a lot of changes (js, unit tests, bok-choy, etc).